Makefile

```diff
@@ -38,12 +38,30 @@ lint: lint.sub
 push: push.sub
 test: test.sub
 
+clean:
+	rm -f .dinit
+	rm -f .buildImage
+	docker rmi -f buildImage > /dev/null 2>&1 || echo -n ""
```

Comment:
yea its the same as "true" but since I totally spaced-out on that and forgot about the easy solution I went with the less obvious one. ;-)  I like 'true' better so its now fixed.